docker/Dockerfile.dev

```diff
@@ -5,6 +5,12 @@ RUN groupadd --gid 10001 app && \
 
 ENV RABBITMQ_CONFIG_FILE=/etc/rabbitmq/rabbitmq
 COPY ./docker/rabbitmq.conf /etc/rabbitmq/rabbitmq.conf
+RUN echo LOG_BASE=/app/workspace/logs > /etc/rabbitmq/rabbitmq-env.conf
+
+# install tini for process management
+ENV TINI_VERSION v0.17.0
```

Comment:
I don't entirely understand why we can't use the version that ships  with Docker, but I asume there's a good reason. Might be nice to add a comment for this.

Reply:
@jgraham Will do. 

The version that ships with docker is only used via the `--init` option of  `docker run`, which is what we were doing before. If we want to customize how tini is called (in our case with `-g`) there's no way to do that with the built-in version, so a common thing to do is to include it in your image.

(Another reason people include it in their images is that the built-in option isn't available with things like docker-compose and other container orchestration software.)